sound/soc/sof/ipc3.c

```diff
@@ -325,8 +325,8 @@ static int sof_ipc3_ctx_ipc(struct snd_sof_dev *sdev, int cmd)
 	struct sof_ipc_reply reply;
 
 	/* send ctx save ipc to dsp */
-	return sof_ipc_tx_message(sdev->ipc, &pm_ctx, sizeof(pm_ctx),
-				  &reply, sizeof(reply));
+	return sof_ipc3_tx_msg(sdev, &pm_ctx, sizeof(pm_ctx),
+			       &reply, sizeof(reply), false);
```

Comment:
this will fit in one line?

Reply:
I try to avoid lines longer than 90 characters, it would be 93, I rather keep it like this. Yes, I know that 100 is the checkpatch limit.